### PR TITLE
media sorts and filters

### PR DESCRIPTION
**Pull request by matthewdias**
feature requests:
[sort](https://kitsu.io/feedback/feature-requests/p/being-able-to-sort-the-anime-page-by-highest-rated)
[filter subtype](https://kitsu.io/feedback/feature-requests/p/sort-in-anime-and-manga-tabs-by-type)
[filter library status](https://kitsu.io/feedback/feature-requests/p/sort-library-by-genreyearetc)

TODO:

~- [ ] sort direction~
~- [ ] filter by library status~
- [x] sort dropdown
- [x] filter by subtype
- [ ] styles

### CODE REVIEW

File: app/templates/media/index.hbs [hunk 1]
```diff
@@ -232,7 +265,28 @@
...
             placeholder=(t "media.index.search")
           }}
-        </div>
+        </span>
+
+        {{! Sort }}
+        <span class="sort-container">
+          {{#bootstrap/bs-dropdown as |dropdown|}}
+            {{#dropdown.button type="button--dark-outline" class="button--small" disabled=text}}
+              {{t (concat 'media.index.sort.' sort)}}
```
Comment (trmcnvn) [inline app/templates/media/index.hbs]: Double quotes :pray:

File: app/templates/media/index.hbs [hunk 2]
```diff
@@ -232,7 +265,28 @@
...
+            {{/dropdown.button}}
+            {{#dropdown.menu}}
+              {{#each sortOptions as |sortOption|}}
+                {{#dropdown.menu-item onClick=(action (mut sort) sortOption)}}
+                  {{t (concat 'media.index.sort.' sortOption)}}
```
Comment (trmcnvn) [inline app/templates/media/index.hbs]: Double quotes :pray:

File: app/templates/media/index.hbs [hunk 3]
```diff
@@ -232,7 +265,28 @@
             update=(action (mut text))
             placeholder=(t "media.index.search")
           }}
-        </div>
+        </span>
+
+        {{! Sort }}
+        <span class="sort-container">
+          {{#bootstrap/bs-dropdown as |dropdown|}}
+            {{#dropdown.button type="button--dark-outline" class="button--small" disabled=text}}
```
Comment (trmcnvn) [inline app/templates/media/index.hbs]: Can we change this to something like `disabled=(not (is-empty text))` so it's a boolean check

**Review by trmcnvn — CHANGES_REQUESTED** [review]

**Comment by matthewdias** [comment]
@JoshFabian think you could style this up as well? should be a quick one.

**Comment by matthewdias** [comment]
moved to #288

**Pull request closed without merging**